src/OpenTelemetry.Api/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## Unreleased
 
+## 1.0.0-RC2
```

Comment:
It seems historically we've been using lowercase (e.g. `1.0.0-rc1`)

Reply:
I'm switch to lowercase to keep consistency..net packages are also lower case